public/index.html

```diff
@@ -0,0 +1,121 @@
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- The Modal -->
+    <div id="myModal" class="modal">
+
+        <!-- Modal content -->
+        <div class="modal-content">
+            <div class="modal-header">
+                <span class="close">&times;</span>
+                <h4>String Class</h4>
+            </div>
+            <div class="modal-body">
+                <span>String Class Methods</span>
+            </div>
+            <div class="modals-footer">
+                <ul>
+              <li><strong>hasVowels</strong><br><span> Returns true if the string contains vowels.</span></li>
+              <li><strong>toUpper</strong><br><span> Returns the String in question but with all characters in upper cases as applicable.</span></li>
+              <li><strong>toLower</strong><br><span > Returns the String in question but with all characters in their lower cases as applicable.</span></li>
+              <li><strong>ucFirst</strong><br><span > Returns the String in question but changes the First Character to an Upper case.</span></li>
+              <li><strong>isQuestion</strong><br><span > Return true if the string is a question (ending with a question mark).</span></li>
+              <li><strong>words</strong><br><span > Returns a list of the words in the string, as an Array.</span></li>
+              <li><strong>wordCount</strong><br><span > Returns the number of words in the string.</span></li>
+              <li><strong>toCurrency</strong><br><span > Returns a currency representation of the String</span></li>
+              <li><strong>fromCurrency</strong><br><span > Returns a number representation of the Currency String </span></li>
+              <li><strong>inverseCase</strong><br><span > Returns each letter in the string as an inverse of its current case e.g Mr. Ben will return mR. bEN.</span></li>
+              <li><strong>alternatingCase</strong><br><span > Returns the letters in alternating cases. It must start with a lower case e.g Onomatopoeia will return oNoMaToPoEiA.</span></li>
+              <li><strong>getMiddle</strong><br><span > Returns the character(s) in the middle of the string</span></li>
+              <li><strong>numberWords</strong><br><span > Returns the numbers in words e.g 325 will return three two five.</span></li>
+              <li><strong>isDigit</strong><br><span > Returns true if the string is a digit (one number) e.g 3 should return true and 34 will return false.</span></li>
+              <li><strong>doubleCheck</strong><span > Returns true if a string contains double characters (including whitespace character).</span></li>
+                </ul>
+                
+            </div>
+        </div>
+
+    </div>
+
+    <footer>
+        <p class="footer">Copyright 2016 - Israel Tomilayo - Andela</p>
+    </footer>
+</body>
+<script>
+// Get the modal
+var modal = document.getElementById('myModal');
```

Comment:
### Nit-Pick
 - Use let statement 
 - I think it is best to separate your script from your view